hack/make-rules/test-integration.sh

```diff
@@ -64,6 +64,9 @@ runTests() {
   kube::log::status "Starting etcd instance"
   CLEANUP_REQUIRED=1
   kube::etcd::start
+  # shellcheck disable=SC2034
+  local ETCD_SCRAPE_PID # Set in kube::etcd::start_scraping, used in cleanup
+  kube::etcd::start_scraping
```

Comment:
The idea of doing this scrape using a simple `curl` for integration tests seems lightweight enough to try.  I'm not entirely sure how you will stitch this back together with which tests were running at a particular time, but given that it doesn't really add to our dependency stack, I'm ok giving it a try.

Reply:
the job stores the etcd logs and the test output with timestamps, it is not great but better than before :shrug:

Reply:
In the case I was looking at, there was a log with timestamps.